testsuite/Gemfile

```diff
@@ -29,7 +30,7 @@ gem 'redis', '~> 5.2'
 gem 'redis-client', '~> 0.22'
 gem 'require_all', '~> 3.0'
 gem 'rubocop', '~>1.66'
-gem 'selenium-webdriver', '4.24'
+gem 'selenium-webdriver', '~> 4'
```

Comment:
Good idea. This way, we are up to date with the chromedriver version we are using that automatically will use the latest version.

Reply:
Haven't we some issues with the latest chromerdiver btw?

Reply:
Not anymore, we were waiting for the latest that included the fix for our issue, and few weeks ago Maxime upgraded it in sumaform, and worked fine.